sdk/managedserviceidentity/Azure.ResourceManager.ManagedServiceIdentities/api/Azure.ResourceManager.ManagedServiceIdentities.netstandard2.0.cs

```diff
@@ -93,6 +135,13 @@ public partial class UserAssignedIdentityResource : Azure.ResourceManager.ArmRes
 }
 namespace Azure.ResourceManager.ManagedServiceIdentities.Models
 {
+    public partial class AssociatedResourceData : Azure.ResourceManager.Models.ResourceData
```

Comment:
We might need a better name for this class

Reply:
renamed